src/opnsense/www/js/opnsense.js

```diff
@@ -205,12 +205,12 @@ function ajaxCall(url, sendData, callback) {
         dataType:"json",
         contentType: "application/json",
         complete: function(data, status) {
-            if ( callback == null ) {
-                null;
-            } else if ( "responseJSON" in data ) {
-                callback(data['responseJSON'],status);
-            } else {
-                callback(data,status);
+            if ( callback != null ) {
+                if ( "responseJSON" in data ) {
```

Comment:
if you are already here, you could use single quotes here

Reply:
Done!